src/ClassicUO.Client/LegionScripting/API.py

```
@@ -0,0 +1,248 @@
+    pass
+
+def ClearIgnoreList():
+    pass
+
+def OnIgnoreList(serial):
+    pass
+
+def Pathfind(x, y, z, distance):
+    pass
+
+def Pathfind(entity, distance):
```

Ruff (0.8.2)

Redefinition of unused `Pathfind` from line 108

(F811)